configuration/config2/EnvironmentLoader.hpp

```diff
@@ -0,0 +1,49 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+
+#ifndef _CONFIG2_ENVIRONMENTLOADER_HPP_
+#define _CONFIG2_ENVIRONMENTLOADER_HPP_
+
+#include <string>
+#include <fstream>
+#include <vector>
+#include "SchemaItem.hpp"
+#include "EnvironmentNode.hpp"
+#include "Status.hpp"
+#include "NameValue.hpp"
+#include "platform.h"
+
+
+class DECL_EXPORT EnvironmentLoader
+{
+    public:
+
+        EnvironmentLoader() { }
+        virtual ~EnvironmentLoader() { }
+        //void addPath(const std::shared_ptr<EnvironmentNode> pNode);
```

Comment:
Not really a fan of keeping commented out code... are these functions going to be readded at some point?

Reply:
Methods were moved during some refactoring. I neglected to remove the commented lines. Lines now removed.